src/foundry/client/data/documents/adventure.d.mts

```diff
@@ -27,6 +27,8 @@ declare global {
    * The client-side Adventure document which extends the common {@link foundry.documents.BaseAdventure} model.
    */
   class Adventure extends ClientDocumentMixin(foundry.documents.BaseAdventure) {
+    // fromSource does not change the parent data signature
```

Comment:
While I've been rethinking my stance on this slowly, we still recommend copying over the signature from the parent to make diffing easier.

You've probably seen some notable cases like `_preCreate` where they were commented out because they were causing mass computation for some reason.